Documentation/project-docs/target-dotnetcore-with-msbuild.md

```diff
@@ -0,0 +1,119 @@
+Using MSBuild to build .NET Core projects
+=========================================
+
+The .NET Core tooling is going to [move from project.json to MSBuild based projects](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/).
+We expect the first version of the .NET Core tools that use MSBuild to ship along with Visual Studio "15".  However, it is possible to use MSBuild for .NET Core
+projects today, and this page shows how.
+
+We recommend that most people targeting .NET Core today use the default tooling experience with project.json.  This is because we haven't yet added support to MSBuild
+for a lot of the benefits that project.json has, because a lot of the ASP.NET based tooling will not work with MSBuild today, and because when we do release .NET Core
+tooling which uses MSBuild, it will be able to automatically convert from project.json to MSBuild based projects.
+
+You may want to consider using MSBuild to target .NET Core for existing projects that already use MSBuild that you want to port to .NET Core, or if you are using
+MSBuild's extensibility in your build for scenarios that are not well supported for project.json projects.
+
+Prerequisites
+=============
+
+- [Visual Studio 2015 Update 2 or higher](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+- [.NET Core tools for Visual Studio](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+- NuGet Visual Studio extension [v3.5.0-beta](https://dist.nuget.org/visualstudio-2015-vsix/v3.5.0-beta/NuGet.Tools.vsix) or later
```

Comment:
I'm not 100% sure but I think that this is needed because netstandard1.5 is not recognized as a framework yet, but using the beta extension will fix this.